### PR TITLE
Release notes for 2.1 

### PR DESCRIPTION
**Pull request by jyaistMap**
@rohitgeo @achapkowski @nanaeaubry @nparavicini7 @rhea-j @priyankatuteja 
- release notes I've compiled as of 1.5.23 - please review and suggest changes that might be needed. This PR needs merged by end of business tomorrow (possibly Monday) to be in place for the developer website release next week

### CODE REVIEW

File: guide/02-api-overview/release_notes_210.ipynb [hunk 1]
```diff
@@ -0,0 +1,897 @@
...
+    "### [Guides](../)\n",
+    "* [Install and Setup](../guide/install-and-set-up/)\n",
+    "  * Adds [`Install in Google Collab`](../guide/install-and-set-up/#install-in-google-colaboratory) section\n",
+    "* [Deep Learning with ArcGIS](../guide/)\n",
+    "  * [2d Computer Vision]\n",
+    "    * [Pixel Classification]\n",
+    "      * [How PSETAE works](../guide/how-PSETAE-works)\n",
+    "  \n",
+    "### [Samples](/python/samples/)\n",
+    "* [GIS analysts and data sciencists](/python/samples)\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:29]: scientists

File: guide/02-api-overview/release_notes_210.ipynb [hunk 2]
```diff
@@ -0,0 +1,897 @@
...
+    "\n",
+    "#### [`arcgis.gis`](/python/api-reference/arcgis.gis.toc.html)\n",
+    "* Adds module so methods can run without specifying backend: `Notebook Server` or `ArcGIS Online`:\n",
+    "  * [`arcgis.notebook`](/python/api-reference/arcgis.notebook.html)\n",
+    "    * [`create_snapshots()`](/python/api-reference/arcgis.notebook.html#create-snapshot)\n",
+    "    * [`execute_notebook()`](/python/api-reference/arcgis.notebook.html#execute-notebook)\n",
+    "    * [`list_runtimes()`](/python/api-reference/arcgis.notebook.html#list-runtimes)\n",
+    "    * [`list_snapshots()`](/python/api-reference/arcgis.notebook.html#list-snapshots)\n",
+    "* Adds Submodule:\n",
+    "  * [`ago.gis.agonb`](/python/api-reference/arcgis.gis.agonb.html)\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:62]: Is this supposed to be arcgis.gis.agonb? Or is there a module named ago

File: guide/02-api-overview/release_notes_210.ipynb [hunk 3]
```diff
@@ -0,0 +1,897 @@
...
+    "    * [`AGOLNotebookManager`](/python/api-reference/arcgis.gis.agonb.html#agolnotebookmanager)\n",
+    "* Adds classes:\n",
+    "  * [`MarketPlaceManger`](/python/api-reference/arcgis.gis.toc.html#marketplacemanager)\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:65]: MarketPlaceManager

File: guide/02-api-overview/release_notes_210.ipynb [hunk 4]
```diff
@@ -0,0 +1,897 @@
...
+    "#### [`arcgis.gis.server`](/python/api-reference/arcgis.gis.server.html)\n",
+    "* [`Service`](/python/api-reference/arcgis.gis.server.html#service)\n",
+    "  * Adds method:\n",
+    "    * [`service_manifest()`](/api-reference/arcgis.gis.server.html#arcgis.gis.server.Service.service_manifest) - result of [**Public Repo Issue #1316**](https://github.com/Esri/arcgis-python-api/issues/1316)\n",
+    "* [`ServicesDirectory`](/python/api-reference/arcgis.gis.server.html#servicesdirectory)\n",
+    "  * Adds method:\n",
+    "    * [`footprints()`](/python/api-reference/arcgis.gis.server.html#arcgis.gis.server.catalog.ServicesDirectory.footprints)\n",
+    "  * Adds ability to work with connections created from `ags` files\n",
+    "* [`SystemManager`](/python/api-reference/arcgis.gis.server.html#systemmanager)\n",
+    "  * Adds preperties:\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:160]: properties

File: guide/02-api-overview/release_notes_210.ipynb [hunk 5]
```diff
@@ -0,0 +1,897 @@
...
+    "  * Adds `extent_type` and `cellsize_type` parameters to following methods:\n",
+    "    * [`majority()`](/python/api-reference/arcgis.raster.toc.html#arcgis.raster.RasterCollection.majority)\n",
+    "    * [`max()`](/python/api-reference/arcgis.raster.toc.html#arcgis.raster.RasterCollection.max)\n",
+    "    * [`mean()`](/python/api-reference/arcgis.raster.toc.html#arcgis.raster.RasterCollection.mean)\n",
+    "    * [`median()`](/python/api-reference/arcgis.raster.toc.html#arcgis.raster.RasterCollection.median)\n",
+    "    * [`min()`](/python/api-reference/arcgis.raster.toc.html#arcgis.raster.RasterCollection.min)\n",
+    "    * [`sum()`](/python/api-reference/arcgis.raster.toc.html#arcgis.raster.RasterCollection.sum)\n",
+    "* [`Raster`](/python/api-reference/arcgis.raster.toc.html#raster)\n",
+    "  * [`multidimensional_info`](/python/api-reference/arcgis.raster.toc.html#arcgis.raster.Raster.multidimensional_info)\n",
+    "    * Adds ability to handle rendering_rule and removes requirment to set the value\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:303]: requirement

File: guide/02-api-overview/release_notes_210.ipynb [hunk 6]
```diff
@@ -0,0 +1,897 @@
...
+    "  * Adds options for the `method` argument:\n",
+    "    * `BAND_WITH_MOST_CHANGE`\n",
+    "    * `SPECTRAL_ANGLE_DISTANCE`\n",
+    "    * `SPECTRAL_EUCLIDEAN_DISTANCE`\n",
+    "\n",
+    "#### [`arcgis.raster.functions.gbl`](/python/api-reference/arcgis.raster.functions.gbl.html)\n",
+    "* Adds `circular_wrap_value` and `circular_calculation` parameters to:\n",
+    "  * [`zonal_statistics()`](/python/api-reference/arcgis.raster.functions.gbl.html#zonal-statistics)\n",
+    "  * [`summarize_raster_within()`](/python/api-reference/arcgis.raster.analytics.html#summarize-raster-within)\n",
+    "  * [`zonal_statistics_as_table)`](/python/api-reference/arcgis.raster.analytics.html#zonal-statistics-as-table)\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:338]: ...as_table()

File: guide/02-api-overview/release_notes_210.ipynb [hunk 7]
```diff
@@ -0,0 +1,897 @@
...
+    "  * Adds parameter table to documentation\n",
+    "   \n",
+    "#### [`arcgis.raster`](/python/api-reference/arcgis.raster.toc.html#)\n",
+    "* [`ImageryLayer`](/python/api-reference/arcgis.raster.toc.html#imagerylayer)\n",
+    "  * Adds methods:\n",
+    "    * [`spectral_profile()`](/api-reference/arcgis.raster.toc.html#arcgis.raster.ImageryLayer.spectral_profile)\n",
+    "  * Adds capability to work with tiles only services to numerous methods:\n",
+    "    * `calculate_volume`\n",
+    "    * `compute_class_stats`\n",
+    "    * `compute_histograms`, \n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:270]: remove comma

File: guide/02-api-overview/release_notes_210.ipynb [hunk 8]
```diff
@@ -0,0 +1,897 @@
...
+    "\n",
+    "#### [`arcgis.raster.functions.gbl`](/python/api-reference/arcgis.raster.functions.gbl.html)\n",
+    "* Adds `circular_wrap_value` and `circular_calculation` parameters to:\n",
+    "  * [`zonal_statistics()`](/python/api-reference/arcgis.raster.functions.gbl.html#zonal-statistics)\n",
+    "  * [`summarize_raster_within()`](/python/api-reference/arcgis.raster.analytics.html#summarize-raster-within)\n",
+    "  * [`zonal_statistics_as_table)`](/python/api-reference/arcgis.raster.analytics.html#zonal-statistics-as-table)\n",
+    "\n",
+    "#### [`arcgis.raster.utils`](/python/api-reference/arcgis.raster.utils.html)\n",
+    "* Adds methods:\n",
+    "  * [`publish_hosted_imagery_layer())`](/python/api-reference/arcgis.raster.utils.html#publish-hosted-imagery-layer)\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:342]: ..._layer()

File: guide/02-api-overview/release_notes_210.ipynb [hunk 9]
```diff
@@ -0,0 +1,897 @@
...
+    "  * [`WebMap`](/python/api-reference/arcgis.mapping.toc.html#webmap)\n",
+    "    * ENH-000134082 Adds methods to transfer layers between `operationalLayers` and `baseMapLayers`\n",
+    "      * [`move_to_basemap()`](/python/api-reference/arcgis.mapping.toc.html#arcgis.mapping.WebMap.move_to_basemap)\n",
+    "      * [`move_from_basemap()`](/python/api-reference/arcgis.mapping.toc.html#arcgis.mapping.WebMap.move_from_basemap)\n",
+    "    * Adds methods:\n",
+    "      * [`update_drawing_info()`](/python/api-reference/arcgis.mapping.toc.html#arcgis.mapping.WebMap.update_drawing_info)\n",
+    "      * [`update_layer()`](/python/api-reference/arcgis.mapping.toc.html#arcgis.mapping.WebMap.update_layer)\n",
+    "    * Adds [`bookmarks`](/python/api-reference/arcgis.mapping.toc.html#arcgis.mapping.WebMap.bookmarks) code sample\n",
+    "    * Adds messaging based on [**Esri Community Post**](https://community.esri.com/t5/arcgis-api-for-python-questions/what-does-the-height-and-width-properties-of-a/m-p/1203164#M7642) to indicate deprecation:\n",
+    "      * [`heighth`](/python/api-reference/arcgis.mapping.toc.html#arcgis.mapping.WebMap.height)\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:387]: height

File: guide/02-api-overview/release_notes_210.ipynb [hunk 10]
```diff
@@ -0,0 +1,897 @@
...
+    "      * `timesteps_of_interest`\n",
+    "      * `channels_of_interest`\n",
+    "      * `n_temporal`\n",
+    "      * `n_temporal_dates`    \n",
+    "* [Automated Machine Learning](/python/api-reference/arcgis.learn.toc.html#automated-machine-learning)\n",
+    "  * [`AutoDL`](/python/api-reference/arcgis.learn.toc.html#autodl)\n",
+    "    * Enhances performance for training\n",
+    "    * Adds options for the models supported for `network` argument\n",
+    "  * [`AutoML`](/python/api-reference/arcgis.learn.toc.html#automl)\n",
+    "    * Adds Shap explanations for each prediction\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:476]: shape?

File: guide/02-api-overview/release_notes_210.ipynb [hunk 11]
```diff
@@ -0,0 +1,897 @@
...
+    "    * `MaskRCNN`\n",
+    "    * `RetinaNet`\n",
+    "    * `MMDetection`\n",
+    "  * [`MaskRCNN`](/python/api-reference/arcgis.learn.toc.html#maskrcnn)\n",
+    "    * Implements test time augmentation for:\n",
+    "      * `fit()`\n",
+    "      * `predict()`\n",
+    "      * `show_results()`\n",
+    "      * `average_precision_score()`\n",
+    "* [Feature, Tabular, anad Timeseries models](/python/api-reference/arcgis.learn.toc.html#feature-tabular-and-timeseries-models)\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:495]: and

File: guide/02-api-overview/release_notes_210.ipynb [hunk 12]
```diff
@@ -0,0 +1,897 @@
...
+    "    * [`from_model()`](/python/api-reference/arcgis.learn.toc.html#aracgis.learn.Pix2PIxHD.from_model)\n",
+    "      * Fixes `TypeError` regarding multiple backbones issue when using a checkpoint file\n",
+    "* [3d Models](/python/api-reference/arcgis.learn.toc/?highlight=pointcloud#d-models)\n",
+    "  * [`RandLANet`](/python/api-reference/arcgis.learn.toc.html#randlanet)\n",
+    "    * [`save()`](/python/api-reference/arcgis.learn.toc.html#arcgis.learn.RandLANet.save)\n",
+    "      * Updates method to improve performance time\n",
+    "* [Image Translation Models](/python/api-reference/arcgis.learn.toc.html#image-translation-models)\n",
+    "  * [`CycleGAN`](/python/api-reference/arcgis.learn.toc.html#cyclegan)\n",
+    "    * Removes erroneous matplotlib warning message  with `show_results()`\n",
+    "* [Feature, Tabular, anad Timeseries models](/python/api-reference/arcgis.learn.toc.html#feature-tabular-and-timeseries-models)\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:850]: and

File: guide/02-api-overview/release_notes_210.ipynb [hunk 13]
```diff
@@ -0,0 +1,897 @@
...
+    "* [`ContentManager`](/python/api-reference/arcgis.gis.toc.html#contentmanager)\n",
+    "  * [`clone_items()`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.ContentManager.clone_items) \n",
+    "    * BUG-000147274 Fixes issue with failing to clone item metadata\n",
+    "    * BUG-000149255 Fixes `KeyError` when cloning Solutions apps\n",
+    "    * BUG-000152758 Fixes issue where layers in a group layer were not being cloned\n",
+    "    * BUG-000153477 Fixes issue where method attempted to clone Living Atlas layers\n",
+    "  * [`advanced_search()`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.ContentManager.advanced_search)\n",
+    "    * Updates to parameter table formatting\n",
+    "  * [`search()`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.ContentManager.search)\n",
+    "    * BUG-000152801 Fixes issue where `categories` paramter was improperly formatted in sent request\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:576]: parameter

File: guide/02-api-overview/release_notes_210.ipynb [hunk 14]
```diff
@@ -0,0 +1,897 @@
...
+    "* [`Group`](/python/api-reference/arcgis.gis.toc.html#group)\n",
+    "  * Fixes [**Public Repo Issue #1279**](https://github.com/Esri/arcgis-python-api/issues/1279) issue where not all properties were being returned when searching so provider value was always returning `None`\n",
+    "* [`ResourceManager`](/python/api-reference/arcgis.gis.toc.html#resourcemanager)\n",
+    "  * [`get()`](/python/api-reference/arcgis.gis.toc.html#arcgis.gis.ResourceManager.get)\n",
+    "    * Fixes return value formatting when `try_jason=False`\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:581]: jason seems like a cool guy, but we should probably try_json

File: guide/02-api-overview/release_notes_210.ipynb [hunk 15]
```diff
@@ -0,0 +1,897 @@
...
+    "* [`enrich()`](/python/api-reference/arcgis.geoenrichment.html#enrich)\n",
+    "  * Fixes `AssertionError` regarding permissions check when connecting to organization with `api_key`\n",
+    "  * Fixes `TypeError` with `NamedArea` as `study_areas` argument\n",
+    "  * Updates documentation to reference details for argument formatting\n",
+    "  * Removes warning that defaults are used when no enrich variables are explicitly entered\n",
+    "  * Updates performance for [`Spatially Enabled Dataframes`]((/python/api-reference/arcgis.features.toc.html#geoaccessor))\n",
+    "  * Fixes issues so `study_areas` argument can be Geometry objects, Dictionaries, or Spatially Enabled DataFrames\n",
+    "  * Fixes failure to return all records when enriching local data\n",
+    "* [`Country()`](/python/api-reference/arcgis.geoenrichment.html#country)\n",
+    "  * Fixes `TypeError` when accessing [`goemetry`](/python/api-reference/arcgis.geoenrichment.html#arcgis.geoenrichment.Country.geometry) property\n",
```
Comment (nparavicini7) [inline guide/02-api-overview/release_notes_210.ipynb:733]: geometry

**Review by nparavicini7 — CHANGES_REQUESTED** [review]
Hey John, I just dug through and looked for typos, as the content seemed good. Apologies for being real nitpicky, but figured I'd just comment on everything I see.

My only question is: are the hyperlinks supposed to be working already? It wouldn't take me to any proper notebook or web location when I clicked on them, and just wanted to make sure that's just a pre-deployment thing and won't still be happening once the site is up and running.

File: guide/02-api-overview/release_notes_210.ipynb [hunk 16]
```diff
@@ -0,0 +1,897 @@
+{
```
Comment (priyankatuteja) [inline guide/02-api-overview/release_notes_210.ipynb:1]: <ul><li><a href="https://github.com/python/samples" target="_blank">Advanced Machine Learning</a></li><li>* 2d Computer Vision</li><li>Object Detection</li><li class="ql-indent-1"><a href="https://github.com/python/samples/detecting-palm-trees-using-deep-learning" target="_blank">Detecting Palm Trees using Deep Learning</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/detecting-swimming-pools-using-automated-machine-learning" target="_blank">Detecting Swimming Pools using Automated Deep Learning</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/traffic-light-detection-on-oriented-imagery" target="_blank">Traffic Light Detection in Oriented Imagery Using ArcGIS Pretrained Model</a></li><li>Feature Extraction</li><li class="ql-indent-1"><a href="https://github.com/python/samples/extracting0-features-and-land-use-land-cover-using-panoptic-segmentation" target="_blank">Extracting features and Land Use Land Cover using Panoptic Segmentation</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/coastline-extraction-usa-landsat8-multispectral-imagery" target="_blank">Coastline extraction using Landsat-8 multispectral imagery and band ratio technique</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/flood-inundation-mapping-using-sar-data-and-deep-learning" target="_blank">Flood inundation mapping and monitoring using SAR data and deep learning</a></li><li>Generative Models</li><li class="ql-indent-1"><a href="https://github.com/python/samples/generating-lst-from-multispectral-imagery-using-pix2pix" target="_blank">Generating Land Surface Temperature from multispectral imagery using Pix2Pix</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/cloud-detector-part1-cloudless-sentinel-&amp;-unsupervised" target="_blank">Generating cloud masks from satellite imagery - Part I</a></li><li><s>Other Imagery Use Cases</s></li><li class="ql-indent-1"><a href="https://github.com/python/samples/cloud-removal-using-pre-trained-deep-learning-model-and-raster-function" target="_blank"><s>Cloud removal using pre-trained deep learning model and raster function</s></a></li><li>Tabular Data Forecasting</li><li class="ql-indent-1"><a href="https://github.com/python/samples/house-price-prediction-using-automl" target="_blank">Property Valuation in King County USA using AutoML and Geoenriched data</a></li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: guide/02-api-overview/release_notes_210.ipynb (hunk 2 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb:62]: Nope. Fixed it.

File: guide/02-api-overview/release_notes_210.ipynb (hunk 16 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb:1]: Changed structure!

File: guide/02-api-overview/release_notes_210.ipynb (hunk 14 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb]: HA! Very clever...I actually lol'd when I read this. Fixed

File: guide/02-api-overview/release_notes_210.ipynb (hunk 13 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb]: Fixed

File: guide/02-api-overview/release_notes_210.ipynb (hunk 15 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb]: Fixed.

File: guide/02-api-overview/release_notes_210.ipynb (hunk 12 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb:847]: Got it.

File: guide/02-api-overview/release_notes_210.ipynb (hunk 11 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb]: Fixed.

File: guide/02-api-overview/release_notes_210.ipynb (hunk 5 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb]: Fixed.

File: guide/02-api-overview/release_notes_210.ipynb (hunk 4 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb]: Fixed.

File: guide/02-api-overview/release_notes_210.ipynb (hunk 3 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb]: Fixed.

File: guide/02-api-overview/release_notes_210.ipynb (hunk 1 above)
Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb:29]: Hey Noah - it's a pre-deployment thing. The structure of the hyperlinks is all through relative links so just running the stand alone notebook most links will not work. Most of them will work if you build the site locally.

Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb:29]: Fixed.

Comment (jyaistMap) [reply guide/02-api-overview/release_notes_210.ipynb]: In addition, the relative links to the api reference are structured in a way that they will not work even when building the site locally. After clicking on the link, uou have to remove the word `python` from the url in order for the link to work in the local site.

**Pull request merged**